rust/cosmos_sdk_grpc_client/README.md

```diff
@@ -3,7 +3,6 @@
 > Rust gRPC client for CØSMOS SDK based blockchains.
 
 [![build](https://github.com/okp4/okp4-cosmos-proto/actions/workflows/build.yml/badge.svg)](https://github.com/okp4/okp4-cosmos-proto/actions/workflows/build.yml)
-[![PyPI](https://img.shields.io/crates/v/cosmos_sdk_grpc_client)](https://crates.io/crates/cosmos_sdk_grpc_client)
```

Comment:
Why do you want to remove this one ?

Reply:
Ah ok. I saw `PyPI` and though it was related to python. Need to change the label instead.

Reply:
@warstrolo we should be good by now.

Reply:
Yes